### DISCUSSION TITLE
xdm青衣值不值得抽啊

### DISCUSSION THREAD

**Discussion by jenemi** [share121/inter-knot#992] [其他]
听说1.3有电强攻，真的假的
![1723259374779](https://github.com/user-attachments/assets/3da01615-cf01-42dd-8643-348ac19600e3)

**Comment by bugmaibear** [comment]
有朱鸢的话我觉得得抽一个（以我浅薄的游戏理解）

**Comment by xibozhi** [comment]
听说电强攻可能是对空六课那个，估计有点强度

**Comment by As52136** [comment]
现在的话性价比就不高了
